### ISSUE TITLE
umlaut handling

### ISSUE THREAD

**Issue by iqualfragile** [spacejam/void#6]
after typing an umlaut the next few keystrokes are not accepted.
to reproduce:
setxkbmap de; void; setxbmap en
in voidmap press type the button right of the L into a label, then try to type something else

**Comment by spacejam** [comment]
hmmm interesting bug, it appears to be something in the underlying library:
```
extern crate termion;                                                                                                                                                                          
                                                                                                                                                                                               
use termion::input::TermRead;                                                                                                                                                                  
use std::io::stdin;                                                                                                                                                                            
                                                                                                                                                                                               
fn main() {                                                                                                                                                                                    
    let stdin = stdin();                                                                                                                                                                       
    for c in stdin.events() {                                                                                                                                                                  
        let evt = c.unwrap();                                                                                                                                                                  
                                                                                                                                                                                               
        println!("got event {:?}", evt);                                                                                                                                                       
    }                                                                                                                                                                                          
}
```

results in similar behavior:
```
λ cargo run
    Finished debug [unoptimized + debuginfo] target(s) in 0.0 secs
     Running `target/debug/termion_bug`
a
got event Key(Char('a'))
got event Key(Char('\n'))
ö
got event Key(Char('ö'))


got event Unsupported

got event Key(Char('\n'))
```

@ticki any idea what may be going on with this?

**Comment by spacejam** [comment]
This has been fixed upstream! Version 0.2.26, just published on crates.io, includes the fix. Thanks for catching and reporting this, @iqualfragile!!!
